srcpkgs/nsxiv/template

```diff
@@ -1,21 +1,20 @@
 # Template file for 'nsxiv'
 pkgname=nsxiv
-version=27.1
+version=28
 revision=1
 build_style=gnu-makefile
```

Comment:
I'm not familiar with void, but nsxiv v28 doesn't use any non-posix/gnu extension in the Makefile. If there is some non posix extension being used, then it should be a bug report on nsxiv repo.

Reply:
No, this is correct.